ldotel/tracing.py

```diff
@@ -25,13 +26,29 @@ class HookOptions:
     """
     If set to true, then the tracing hook will add the evaluated flag value to
     span events.
+
+    .. deprecated:: 1.0.0
+        This option is deprecated and will be removed in a future version.
+        Use :attr:`include_value` instead.
+    """
+
+    include_value: bool = False
+    """
+    If set to true, then the tracing hook will add the evaluated flag value to
+    span events.
     """
 
 
 class Hook(LDHook):
     def __init__(self, options: HookOptions = HookOptions()):
         self.__tracer = trace.get_tracer_provider().get_tracer("launchdarkly")
         self.__options = options
+        if self.__options.include_variant:
```

Comment:
I don't know an actually reasonable way to do this without the logger instance.

Reply:
You can just grab the logger from wherever. Python is just like that.

```python
from logging import getLogger

logger = getLogger('launchdarkly-otel')
logger.warning('Hi there')
```

If we document that we always use the 'launchdarkly-otel' named logger, then it makes it easy for customers to affect the formatting without having to pass a logger around everywhere.

But this honestly seems pretty reasonable to me.